cmd/cli/plugin/package/package_available_get.go

```diff
@@ -156,3 +169,50 @@ func getValuesSchemaForPackage(namespace, name, version string, kc kappclient.Cl
 
 	return nil
 }
+
+func generateDefaultValuesForPackage(pkg *kapppkg.Package, valuesFile ...*os.File) error {
+	if pkg == nil {
+		// should not happen
+		return errors.New("pkg is nil")
+	}
+
+	if len(pkg.Spec.ValuesSchema.OpenAPIv3.Raw) == 0 {
```

Comment:
question: will there be an nil exception when `ValueSchema` does not exist in `Spec` ?

Reply:
If `ValuesSchema` is empty when creating a package CR it gets defaulted to the following and `len(pkg.Spec.ValuesSchema.OpenAPIv3.Raw) == 0` is true. Verified with a dummy package.
```
  valuesSchema:
    openAPIv3: null
```